app/views/devise/sessions/new.html.slim

```diff
@@ -1,25 +1,28 @@
-h2
-  | Log in
-
-= render 'shared/notifications'
```

Comment:
My take on this would be to first have the error messages and then improve them rather than omitting them until we get them properly designed.